eras/dijkstra/impl/src/Cardano/Ledger/Dijkstra/TxBody.hs

```diff
@@ -1058,10 +1130,22 @@ guardsDijkstraTxBodyRawL =
         x@DijkstraSubTxBodyRaw {} -> \y -> x {dstbrGuards = y}
     )
 
-instance DijkstraEraTxBody DijkstraEra where
+subTransactionsDijkstraTxBodyL :: Lens' (DijkstraTxBodyRaw TopTx era) (OMap TxId (Tx SubTx era))
+subTransactionsDijkstraTxBodyL = lens dtbrSubTransactions (\x y -> x {dtbrSubTransactions = y})
+
+instance EraTx DijkstraEra => HasOKey TxId (Tx SubTx DijkstraEra) where
+  toOKey = txIdTx
```

Comment:
It will be better to move this instance to `cardano-ledger-core`, so we don't have to redefine it for every era:
```haskell
instance EraTx era => HasOKey TxId (Tx l era) where
  toOKey = txIdTx
```

But, we don't need to block this PR, because of that.

Reply:
Created a ticket for it, so we don't forget: #5399